client/src/main/resources/org/schlunzis/kurtama/client/fx/controller/css/style.css

```diff
@@ -116,3 +116,29 @@ TextField:focused, PasswordField:focused {
 .label {
     -fx-text-fill: k-text-color;
 }
+
+/*
+ ComboBox
+ */
+ComboBox {
+    -fx-background-color: k-textfield-focused;
+}
+
+ComboBox .arrow {
+    -fx-background-color: k-text-color;
+}
+
+/*
+ TextArea
+ */
+
+TextArea {
+    -fx-font-family: monospace;
```

Comment:
Do we want all TextAreas to have a monospace font or just the log area? If the latter, we should consider doing this with a css class instead of the whole TextArea element.